src/ImageBuilder/README.md

```diff
@@ -47,23 +47,25 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
+branch: 4b4bb1021353692578499f43f1aa912964a2b7e2
 require:
   - $(this-folder)/../readme.azure.noprofile.md
 input-file:
-  - $(repo)/specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json
-branch: 9f3ac7b135ac83007b7f3f68ca8ca9705284cff9
+  - $(repo)/specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-07-01/imagebuilder.json
+
 title: ImageBuilder
-module-version: 0.1.0
+module-version: 0.2.0
 subject-prefix: $(service-name)
 
 identity-correction-for-post: true
 resourcegroup-append: true
+nested-object-to-string: true
 
 directive:
   # 1. Remove the unexpanded parameter set
   # 2. For New-* cmdlets, ViaIdentity is not required, so CreateViaIdentityExpanded is removed as well
   - where:
-      variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$
+      variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$|^Update$|^UpdateViaIdentity$
```

Comment:
as we discussed, if you want to remove variants that are not expanded, use more precise regex

Reply:
I will re-check the readme document.

Reply:
Done